.github/workflows/cpack.yml

```diff
@@ -15,7 +15,9 @@ jobs:
   package:
     runs-on: ubuntu-20.04
     # optionally override entire container image:tag string
-    container: ${{ matrix.distro.container || format('{0}:{1}', matrix.distro.name, matrix.distro.version) }} 
+    container: 
+      image: ${{ matrix.distro.container || format('{0}:{1}', matrix.distro.name, matrix.distro.version) }}
+      options: --platform=${{ matrix.arch.cmake }} 
```

Comment:
the platform argument must match the format `os[/arch[/variant]]`. e.g. `linux/aarch64`

https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate

Reply:
I haven't yet found a value other than `amd64` and `linux/amd64` that works for `--platform`. The symptom when it's not working is the container stops running before the first job step runs.